package.json

```diff
@@ -61,14 +61,16 @@
     "eslint-config-ca": "^1.0.0",
     "flow-bin": "^0.36.0",
     "flow-coverage-report": "^0.2.0",
+    "html-webpack-plugin": "^2.28.0",
```

Comment:
`html-webpack-plugin` should be in the `dependencies`, not `devDependencies`.

`dependencies`: libraries that the user of `ca-webpack-config` needs to be able to run webpack on their project.
`devDependencies`:  librairies tha are needed to build `ca-webpack-config` and generate the `lib` folder.